src/patternfly/components/Alert/alert.scss

```diff
@@ -87,6 +119,30 @@
     --pf-c-alert__icon--Color: var(--pf-c-alert--m-info__icon--Color);
     --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--m-info__icon--BackgroundColor);
     --pf-c-alert__title--Color: var(--pf-c-alert--m-info__title--Color);
+
+    // Set border left color in case it is also an inline alert
+    --pf-c-alert--m-inline--BorderLeftColor: var(--pf-c-alert--m-info-inline--BorderLeftColor);
+  }
+
+  &.pf-m-inline {
+    --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
+    --pf-c-alert__title--Color: var(--pf-c-alert--m-inline__title--Color);
+    --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--BackgroundColor);
+    --pf-c-alert--m-success__icon--Color: var(--pf-c-alert--m-success-inline__icon--Color);
+    --pf-c-alert--m-danger__icon--Color: var(--pf-c-alert--m-danger-inline__icon--Color);
+    --pf-c-alert--m-warning__icon--Color: var(--pf-c-alert--m-warning-inline__icon--Color);
+    --pf-c-alert--m-info__icon--Color: var(--pf-c-alert--m-info-inline__icon--Color);
+
+    border-color: var(--pf-c-alert--m-inline--BorderColor);
+    border-style: var(--pf-c-alert--m-inline--BorderStyle);
+    border-width: var(--pf-c-alert--m-inline--BorderWidth);
+    border-left-color: var(--pf-c-alert--m-inline--BorderLeftColor);
```

Comment:
You can write this in shorthand to save some lines. We also typically don't define `border-style` as a var, but you can if you want.

That said how about we define this border as a pseudo element instead? The triangle corner is somewhat obvious.

<img width="94" alt="Screen Shot 2019-04-29 at 1 12 57 PM" src="https://user-images.githubusercontent.com/35148959/56917404-03e25200-6a81-11e9-8382-fce0c369cdc0.png">